.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Create Release Pull Request'
+
+on:
+  create:
+    branches:
+      - releases/*
+
+jobs:
+  build:
+    name: Create Release Pull Request
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v2
+        if: ${{ !contains(github.ref, 'test') }}
+
+      - name: Create Pull Request content
+        if: ${{ !contains(github.ref, 'test') }}
+        run: |
+          PR_TITLE=`./script/draft-release/release-pr-content.sh title ${GITHUB_REF#refs/heads/}`
+          PR_BODY=`./script/draft-release/release-pr-content.sh body ${GITHUB_REF#refs/heads/}`
+
+          echo "PR_BODY<<EOF" >> $GITHUB_ENV
+          echo "$PR_BODY" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+          echo "PR_TITLE<<EOF" >> $GITHUB_ENV
+          echo "$PR_TITLE" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        if: ${{ !contains(github.ref, 'test') }}
+        with:
+          app_id: ${{ secrets.DESKTOP_RELEASES_APP_ID }}
+          private_key: ${{ secrets.DESKTOP_RELEASES_APP_PRIVATE_KEY }}
+
+      - name: Create Release Pull Request
+        uses: peter-evans/create-pull-request@v4.0.4
+        if: ${{ !contains(github.ref, 'test') }}
+        with:
```

Comment:
Maybe we should add `draft: true` until the creator of the PR checks everything is ok? (Sentry errors, stats and feature flags)

Reply:
I like the idea of draft: true; Might not hurt to update the releasing updates readme to something of the effect of -> After you have completed the release pr checklist, update it from draft to ready for review.